service/hook/common/common.go

```diff
@@ -42,7 +42,9 @@ type TransformResponseModel struct {
 // TransformResponseInputModel ...
 type TransformResponseInputModel struct {
 	// Errors include the errors if the build could not trigger
-	Errors []string
+	Errors                  []string
+	SkippedTriggerResponses []bitriseapi.SkipAPIResponseModel
```

Comment:
Add comment - like the one `SuccessTriggerResponses` has and move it to be the last one, after `FailedTriggerResponses`